src/processing/2_Verden/Verden.md

```diff
@@ -0,0 +1,492 @@
+---
+title: Verden
+level: 2
+author: Kine Gjerstad Eide
+language: nb
+---
+
+# Intro {.intro}
+Denne oppgava bygger på oppgava med samme navn som ligger på introduksjonsnivå her i Processingoppgavene. 
```

Comment:
du må gjerne linke til oppgaven. det kan du gjøre ved å skrive noe tekst inni firkantparanteser: [Klikk her for å se oppgaveteksten.], etterfult av (../1_Verden/verden.html){target=_blank} bak.